src/main.rs

```diff
@@ -106,9 +153,14 @@ fn set_priv_to_machine() {
 
 fn enable_risc_interrupts() {
     unsafe {
-        riscv::register::mie::set_mtimer();
+        // riscv::register::mie::set_mtimer();
+        riscv::register::mie::set_mext();
```

Comment:
It looks like this & `dr.core_peripherals.plic.mext.enable();` down in `kmain` are duplicates of each other: the implementation I see for the latter calls `mie::set_mext();`

Reply:
oh fascinating, true.